docs/ref/templates/builtins.txt

```diff
@@ -515,15 +516,6 @@ of how ``x in y`` will be interpreted::
 
 Not contained within. This is the negation of the ``in`` operator.
 
-The comparison operators cannot be 'chained' like in Python or in mathematical
```

Comment:
Actually it probably makes sense to put it as a paragraph after "if` tags may also use..." before the list of all the operators. In that case, an admonition box probably isn't needed.